pandas/core/computation/ops.py

```diff
@@ -14,6 +15,7 @@
 import pandas as pd
 from pandas.core.base import StringMixin
 import pandas.core.common as com
+from pandas.core.computation.check import _NUMEXPR_INSTALLED, _NUMEXPR_VERSION
```

Comment:
@anjsudh ok, the problem is we can't import here (we have a check in code_checks to avoid importing the optional dependencies until they are needed).

So. I think just add floor & ceil to the _math_ops (always) and better to do a run-time check if this op exists in ne (not exactly sure how to do that).